src/utils/components/CPUMemoryModal/CPUMemoryModal.tsx

```diff
@@ -138,7 +138,7 @@ const CPUMemoryModal: FC<CPUMemoryModalProps> = ({
         variant={AlertVariant.info}
       />
       <div className="inputs">
-        <CPUInput cpu={cpu} setCPU={setCPU} />
+        <CPUInput cpu={cpu} setCPU={setCPU} vmCPU={getCPU(vm)} />
```

Comment:
It's hard to understand why we need both cpu and vmCPU props, is one for the input and the second for current VM's CPU? Maybe we need better names to make it more intuitive

Reply:
`cpu` is the state variable to track what the user has entered. `vmCPU` is what's currently set on the VM/template VM spec. It's used to determine the defaults for the 'Restore defaults' button. Maybe I should rename it to `defaultCPU`?

Reply:
defaultCPU does sound better, I thought maybe cpu could be called `cpuInput` and you can keep `vmCPU` or rename it to `defaultCPU` whichever sounds better to you